src/graph.py

```diff
@@ -635,6 +642,59 @@ def grid_search(self, prize_file, Gs, Bs, Ws):
         return forest, augmented_forest, params_by_nodes
 
 
+    def _eval_randomizations(self, params):
+        """
+        Convenience method which sets parameters and performs PCSF randomizations.
+
+        Arguments: 
+            params (dict): params with which to run the program
+
+        Returns: 
+            str: Parameter values in string format 
+            networkx.Graph: forest
+            networkx.Graph: augmented_forest
+        """
+
+        self._reset_hyperparameters(params=params)
+        paramstring = "W_{:04.2f}_B_{:04.2f}_G_{:d}".format(self.params.w, self.params.b, self.params.g)
+        logger.info("Randomizations for " + paramstring)
+
+        forest, augmented_forest = self.randomizations(params["noisy_edge_reps"], params["random_terminals_reps"])
+        
+        return paramstring, forest, augmented_forest
+
+
+    def _grid_randomization(self, prize_file, Ws, Bs, Gs, noisy_edges_reps=0, random_terminals_reps=0):
```

Comment:
Why does this method have a leading `_`?

Reply:
075e1a45433dcde1858b5ab0fbe3db0539818962